components/uniset/src/provider.rs

```diff
@@ -656,38 +657,37 @@ pub mod key {
 
 #[derive(Debug, Hash, Eq, PartialEq, Clone)]
 #[cfg_attr(feature = "provider_serde", derive(serde::Serialize, serde::Deserialize))]
-pub struct UnicodeProperty<'s> {
+pub struct UnicodePropertyV1<'s> {
     pub name: Cow<'s, str>,
-    pub inv_list: Vec<u32>,
+    pub inv_list: UnicodeSet,
```

Comment:
Nit optional: consider changing the key name to reflect that this is now structured data (something like "set", etc.). Or ignore this suggestion.